examples/SLAM-camera/js/hit-test-location-root.ts

```diff
@@ -22,94 +22,91 @@ export class HitTestLocationRoot extends Component {
     @property.object()
     camera!: WLEObject;
 
-    private _tempScaling = new Float32Array(3);
-    private _visible = false;
+    tempScaling = new Float32Array(3);
```

Comment:
Either declare it outside of the class (global to the file), or use `_` to indicate private